src/games/snake/snake.py

```diff
@@ -1,6 +1,117 @@
-from src.games.game import *
+from pygame import sprite
+from src.games.snake.snake_part import SnakePart
+from src.resources.snake_game_resources import SnakeGameResources
+from src.settings import SnakeSettings
 
 
-class Snake(Game):
-    def __init__(self, level, is_sound_on):
-        Game.__init__(self, level, is_sound_on)
+class Snake():
+    def __init__(self, speed, board, is_sound_on):
+        self._direction = SnakeSettings.SNAKE_INITIAL_DIRECTION
+        self._prev_direction = self._direction
+        self._isAlive = True
+        self._pointsOccupied = sprite.Group()
+        self._head = SnakeSettings.SNAKE_INITIAL_POSITION
+        self._snake_speed = speed
+        self._time_since_last_update = self._snake_speed
+        self._board = board
+        self._is_sound_on = is_sound_on
+        self._create_body_part() # makes head of snake
+
+
+    def _create_body_part(self, center_of_rect = None):
```

Comment:
In the default value you shouldn't use spaces:
```suggestion
    def _create_body_part(self, center_of_rect=None):
```